README.md

```diff
@@ -1 +1,12 @@
-# proposals
\ No newline at end of file
+# Stencil Community Proposals
+
+Use this repository to create proposals for new Stencil Community packages.
+
+## Usage
+
+Proposals are written in issues.
+
+To create a new proposal,
+
+1. Search for existing proposals in [the issues](https://github.com/stencil-community/proposals/issues). The proposal may exist already. If it does, consider reading through it and adding your thoughts. You can also react to proposals by adding a :+1: reaction.
```

Comment:
I'd consider using slightly more forceful language here to prevent duplicate proposals:
```suggestion
1. Search for existing proposals in [the issues](https://github.com/stencil-community/proposals/issues). The proposal may exist already. If it does, please feel free to add a :+1: reaction to the issue summary, and/or by adding your thoughts in the form of a comment on the issue. 
```

Reply:
Fixed in 1064b9ad027e7d351719ede1ba43eb19ac9dc3a8